snap-guest

```diff
@@ -226,9 +226,7 @@ qemu-img create -f qcow2 -b $SOURCE_IMG $TARGET_IMG
 # -- Image Manipulation ----------
 
 if [ $CLOUD -eq 0 -o $CLOUD_PRECONF -eq 1 ]; then
-  export COMMAND TARGET_HOSTNAME MAC SOURCE_NAME SWAP TARGET_IMG TARGET_NAME \
-    STATIC_IPADDR STATIC_NETMASK STATIC_GATEWAY
```

Comment:
this part was causing race condition on multiple snap-guest instances (they touch the common environment and overwrite the variables)